### ISSUE TITLE
[Warning!] Kinesis doesn't take seriously about burning users eyes!

### ISSUE THREAD

**Issue by adofse** [KinesisCorporation/Adv360-Pro-ZMK#604]
I opened the following two issues but Kinesis Corporation was reluctant to state the official statement and close those issues without responding for like one month!
- https://github.com/KinesisCorporation/Adv360-Pro-ZMK/issues/560
- https://github.com/KinesisCorporation/Adv360-Pro-ZMK/issues/570

Those issues are related to harming users eyes so it should not be ignored!
At least provide WHY you can't fix it or TIMELINE to fix it.

So disappointed how Kinesis Corporation treats precious user's feedback.
They want fix this complaint by just RMA😂 It's easy for them by just not fixing it. That's a shame.

**Comment by danielleevandenbosch** [comment]
What are you referring to?

**Comment by aaronjensen** [comment]
@adofse Have you tried adjusting `CONFIG_ZMK_RGB_UNDERGLOW_BRT_MAX` and/or `CONFIG_ZMK_RGB_UNDERGLOW_BRT_START` to something less than `100`? I haven't tried it, just came across it.

**Comment by aaronjensen** [comment]
FWIW, I don't think this works. I don't know that the LEDs have brightness control. I wasn't able to get it to work in my limited testing.

**Comment by terids** [comment]
I have to agree that the LEDs burn the retinas when activating the boot-loader. You have to be looking almost directly at them when pushing a paperclip in. I always end up with a dead spot in my vision for around 5 minutes after.
Normal operation is totally fine.
